src/packages/solid/components/Tile/Artwork.styles.ts

```diff
@@ -0,0 +1,58 @@
+import theme from 'theme';
+import { Color, StateStyle } from '../../../../shared/types/solid';
+import { getHexColor } from '../../../../shared/utils/index'; // TODO ts path aliasing
+
+type ArtworkStyle = {
+  Container: {
+    // animationBlurEntrance: object;
```

Comment:
do we need to keep all of the commented out values in here for the future or can they be removed?